### PR TITLE
Added new test which checks placement with optimal partitioning

### PR DESCRIPTION
**Pull request by VVVyacheslav**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/series/SeriesQueryPlacementOptimalPartitioningTest.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        }
+
+        assertEquals(actualGroupSetOfSeries, expectedGroupSetOfSeries, "The sets of grouped series do not match expected.");
+    }
+
+    @Issue("5965")
+    @Test(dependsOnMethods = "testOfGroupCount", description = "Checks that parameter totalScore is correct")
+    public void testOfTotalScore() {
+        List<Series> seriesList = querySeriesAsList(QUERY);
+        double actualTotalScore = seriesList.get(0).getGroup().getTotalScore().doubleValue();
```
Comment (raipc) [inline src/test/java/com/axibase/tsd/api/method/series/SeriesQueryPlacementOptimalPartitioningTest.java]: Check list size

File: src/main/java/com/axibase/tsd/api/model/series/query/transformation/group/Place.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class Place {
+    private int count;
```
Comment (raipc) [inline src/main/java/com/axibase/tsd/api/model/series/query/transformation/group/Place.java]: Can't the fields be final?

**Comment by MikhailZvagelsky** [comment]
Ok :)

File: src/test/java/com/axibase/tsd/api/method/series/SeriesQueryPlacementOptimalPartitioningTest.java [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+        int seriesCount = 5;
+        Series[] seriesArray = new Series[seriesCount];
+        for (int i = 0; i < seriesCount; i++) {
+            seriesArray[i] = new Series(String.format("lp_%s", i + 1), METRIC_NAME);
+        }
+
+        long totalSamplesCount = 10;
+
+        for (int i = 0; i < totalSamplesCount; i++) {
+            final List<Double[]> value = new ArrayList<>(Arrays.asList(
```
Comment (raipc) [inline src/test/java/com/axibase/tsd/api/method/series/SeriesQueryPlacementOptimalPartitioningTest.java]: Rename to `seriesValuesToInsert`, move to the beginning of the method. Use `double[][]` datatype

File: src/test/java/com/axibase/tsd/api/method/series/SeriesQueryPlacementOptimalPartitioningTest.java [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+    private static final double EXPECTED_GROUP_SCORE_SECOND = 6.0;
+
+    private static final SeriesQuery QUERY = new SeriesQuery(QUERY_ENTITY, METRIC_NAME, START_DATE, END_DATE)
+            .setGroup(new Group()
+                    .setType(GroupType.SUM)
+                    .setPlace(PLACE));
+
+    @BeforeClass
+    private void insertSeries() throws Exception {
+        int seriesCount = 5;
```
Comment (raipc) [inline src/test/java/com/axibase/tsd/api/method/series/SeriesQueryPlacementOptimalPartitioningTest.java]:
```java
final int seriesCount = seriesValuesToInsert.length()
```

File: src/test/java/com/axibase/tsd/api/method/series/SeriesQueryPlacementOptimalPartitioningTest.java [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+        Series[] seriesArray = new Series[seriesCount];
+        for (int i = 0; i < seriesCount; i++) {
+            seriesArray[i] = new Series(String.format("lp_%s", i + 1), METRIC_NAME);
+        }
+
+        long totalSamplesCount = 10;
```
Comment (raipc) [inline src/test/java/com/axibase/tsd/api/method/series/SeriesQueryPlacementOptimalPartitioningTest.java]:
```java
final int totalSamplesCount = seriesValuesToInsert[0].length();
```

**Pull request merged**